spec/shoryuken/processor_spec.rb

```diff
@@ -299,7 +299,7 @@ def perform(sqs_msg, body); end
       it 'does not extend the message invisibility' do
         expect(sqs_msg).to receive(:visibility_timeout=).never
         expect_any_instance_of(TestWorker).to receive(:perform).and_raise 'worker failed'
-        expect { subject.process(queue, sqs_msg) }.to raise_error
+        expect { subject.process(queue, sqs_msg) }.to raise_error(RuntimeError, "worker failed")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.